### DISCUSSION TITLE
When will we get a Jakarta-compatible Quartz release?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [quartz-scheduler/quartz#1187] [Q&A] [answered]
I see two release candidates for 2.4, and one for 2.5.  Are there any releases compatible with Jakarta?

**Comment by jhouserizer** [comment, marked as answer]
Can't promise a date, but soon something like 2-14 days

**Reply by indyaah** [reply]
@jhouserizer is there any place where I can lend help with this effort?

**Comment by mprins** [comment]
https://github.com/quartz-scheduler/quartz/releases/tag/v2.5.0

**Comment by ed-erwin-tf** [comment]
Thanks for the 2.5 release.

After updating to 2.5.0 I have found that I need to add a runtime dependency on jakarta.transaction-api.  I hadn't needed that before.  I don't see it listed as a required dependency in the pom, but maybe it should be.

**Reply by jacobdev2** [reply]
How exactly did you do this? I'm running into a problem where it works if the jobs are just in RAM, but as soon as I go to the database, the beans fail to create.

**Reply by jacobdev2** [reply]
For me, it was this:

		<dependency>
		    <groupId>com.mchange</groupId>
		    <artifactId>c3p0</artifactId>
		    <version>0.10.1</version>
		    <scope>runtime</scope>
		</dependency>
